### PR TITLE
IN-29 feat: add Icon component, icon.stories, delete unused files

### PR DESCRIPTION
**Pull request by MaksimSinicbIn**

### CODE REVIEW

File: src/components/icon/icon.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import React from "react"
+
+import { Icon } from "./icon"
+
+const meta = {
+  component: Icon,
+  tags: ["autodocs"],
+  title: "Components/Icon"
```
Comment (vitaliiRss) [inline src/components/icon/icon.stories.tsx:10]: Please change this line `title: "01. Design System/Icons"`

File: src/components/icon/icon.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+        gap: "5px",
+        justifyContent: "space-around",
+        margin: "auto",
+        padding: "10%",
+        width: "50%"
+      }}
+    >
+      {iconsItems.map(i => {
+        return (
+          <Icon height={"34"} iconId={i.iconId} key={i.iconId} viewBox={"0 0 34 34"} width={"34"} />
```
Comment (vitaliiRss) [inline src/components/icon/icon.stories.tsx:273]: Could you please explain why you add custom values for viewBox, width and height? I guess default values is enough for preview in storybook

Comment (vitaliiRss) [inline src/components/icon/icon.stories.tsx:273]: Can you add html structure for preview all icons with small css to make it more attractive?
```
<div style={some styles}>
  map => <div style={some styles}><Icon ... /></div>
</div>
```

File: src/components/icon/icon.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+
+const meta = {
+  component: Icon,
+  tags: ["autodocs"],
+  title: "Components/Icon"
+} satisfies Meta<typeof Icon>
+
+export default meta
+
+type Story = StoryObj<typeof Icon>
```
[the remark this answers is not in the document]
Comment (MaksimSinicbIn) [reply src/components/icon/icon.stories.tsx:15]: fixed

File: src/components/icon/icon.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+
+const iconsItems = [
+  {
+    iconId: "home-outline"
+  },
+  {
```
[the remark this answers is not in the document]
Comment (MaksimSinicbIn) [reply src/components/icon/icon.stories.tsx:21]: fixed

File: src/components/icon/icon.stories.tsx (hunk 1 above)
Comment (MaksimSinicbIn) [reply src/components/icon/icon.stories.tsx:10]: fixed

File: src/components/icon/icon.stories.tsx (hunk 2 above)
Comment (MaksimSinicbIn) [reply src/components/icon/icon.stories.tsx]: Fixed

File: src/components/icon/icon.stories.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MaksimSinicbIn) [reply src/components/icon/icon.stories.tsx:15]: In this case, in order not to complicate working with Storybook, this typification is used to draw an array of elements.

**Review by vitaliiRss — APPROVED** [review]

**Review by web-serge — APPROVED** [review]

**Pull request merged**